cms/celery.py

```diff
@@ -21,3 +19,18 @@
 # pickle the object when using Windows.
 APP.config_from_object('django.conf:settings')
 APP.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+class Router(AlternateEnvironmentRouter):
+    """
+    An implmentation of AlternateEnvironmentRouter, for routing tasks to non-cms queues.
+    """
+
+    @property
+    def alternate_env_tasks(self):
```

Comment:
Nit: Is there a reason this needs to be a property on the instance object?  Generally, in python, we just specify this as a constant variable on the class since it's class-specific and not instance-specific.  
I am keeping this as a Nit comment since it doesn't affect the functionality.